addons/hr_fleet/models/employee.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import io
+
+from odoo import api, fields, models
+
+class Employee(models.Model):
+    _inherit = 'hr.employee'
+
+    employee_cars_count = fields.Integer(compute="_compute_employee_cars_count", string="Cars")
+
+
+    def action_open_employee_cars(self):
+        self.ensure_one()
+        Fleet = self.env['fleet.vehicle.assignation.log']
+        car_line = Fleet.search([('driver_id','=',self.user_id.partner_id.id)]).mapped('vehicle_id')
+
+        return {
+            "type": "ir.actions.act_window",
+            "res_model": "fleet.vehicle",
+            "views": [[False, "kanban"], [False, "form"], [False, "tree"]],
+            "domain": [("id", "in", car_line.ids)],
+            "name": "History Employee Cars",
+        }
+
+    def _compute_employee_cars_count(self):
+        Fleet = self.env['fleet.vehicle.assignation.log'].sudo()
+        driver_id = self.mapped('user_id').mapped('partner_id').ids
+        fleet_data = Fleet.read_group(domain=[('driver_id', 'in', driver_id)], fields=['driver_id'], groupby=['driver_id'])
+        mapped_data = dict([(m['driver_id'][0], m['driver_id_count']) for m in fleet_data])
+        for employee in self:
+            employee.employee_cars_count = mapped_data.get(employee.user_id.partner_id.id)
```

Comment:
It's great you used ``read_group`` instead of ``search_count``. This is a good practice!